configs/system/base.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
   group: null,
   core: {
     docker: {
-      image: 'dashpay/dashd:0.17.0.0-rc5',
+      image: 'dashpay/dashd:0.17.0.3',
```

Comment:
I would recommend just setting the `0.17` tag so users can do `dashmate update` instead of releasing dashmate for every minor Core release.